.github/actions/kfp-cluster/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "Set up KFP on KinD"
+description: "Step to start and configure KFP on KinD"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create k8s Kind Cluster
+      uses: container-tools/kind-action@v2
+      with:
+        cluster_name: kfp
+        kubectl_version: v1.29.2
+        version: v0.22.0
+        node_image: kindest/node:v1.29.2
+
+    - name: Build images
+      shell: bash
+      run: ./scripts/deploy/github/build-images.sh
+
+    - name: Deploy KFP
+      shell: bash
+      run: ./scripts/deploy/github/deploy-kfp.sh
```

Comment:
Looks like this script deploys kfp-tekton and not pipelines. Could you update to add kfp deploy steps here instead?

Reply:
Fixed in https://github.com/kubeflow/pipelines/pull/10946.
This PR must be rebased and merged after https://github.com/kubeflow/pipelines/pull/10946.

Reply:
I'm reverting it to draft.